go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/h2non/filetype v1.0.10
 	github.com/h2non/go-is-svg v0.0.0-20160927212452-35e8c4b0612c
+	github.com/hashicorp/go-version v1.2.0
```

Comment:
The server uses https://github.com/blang/semver for parsing version. 1/5 that we should to the same here.